pallets/allocations/src/weights.rs

```diff
@@ -57,106 +57,68 @@ pub trait WeightInfo {
 
 /// Weights for pallet_allocations using the Substrate node and recommended hardware.
 pub struct SubstrateWeight<T>(PhantomData<T>);
+
 impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
-	// Storage: Allocations SessionQuota (r:1 w:1)
-	// Storage: Balances TotalIssuance (r:1 w:1)
-	// Storage: System Account (r:3 w:3)
-	// Storage: System Number (r:1 w:0)
-	// Storage: System ExecutionPhase (r:1 w:0)
-	// Storage: System EventCount (r:1 w:1)
-	// Storage: System Events (r:1 w:1)
-	/// The range of component `b` is `[1, 500]`.
 	fn allocate(b: u32) -> Weight {
-		Weight::from_ref_time(73_910_000 as u64)
-			// Standard Error: 9_000
-			.saturating_add(Weight::from_ref_time(26_411_000).saturating_mul(b as u64))
-			.saturating_add(T::DbWeight::get().reads(8 as u64))
-			.saturating_add(T::DbWeight::get().reads((1 as u64).saturating_mul(b as u64)))
-			.saturating_add(T::DbWeight::get().writes(6 as u64))
-			.saturating_add(T::DbWeight::get().writes((1 as u64).saturating_mul(b as u64)))
+		Weight::from_ref_time(0_u64)
+			// Standard Error: 1_500_000
```

Comment:
This standard error which had increased significantly may be telling us something.